src/api/comments.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import inatjs from "inaturalistjs";
+
+import handleError from "./error";
+
+const createComments = async (
```

Comment:
This creates a comment, not multiple comments, so I'd prefer `createComment`. Same elsewhere.